__main__.py

```diff
@@ -118,15 +119,37 @@ def init_logging(args):
 def init(args):
     global CONFIG, binaryDownload
 
+    if len(CONFIG.get_value('files-path')) == 0:
+        CONFIG.set_value('files-path', os.path.abspath('files'))
+    if len(CONFIG.get_value('crackers-path')) == 0:
+        CONFIG.set_value('crackers-path', os.path.abspath('crackers'))
+    if len(CONFIG.get_value('hashlists-path')) == 0:
+        CONFIG.set_value('hashlists-path', os.path.abspath('hashlists'))
+    if len(CONFIG.get_value('zaps-path')) == 0:
+        CONFIG.set_value('zaps-path', os.path.abspath('.'))
+    if len(CONFIG.get_value('preprocessors-path')) == 0:
+        CONFIG.set_value('preprocessors-path', os.path.abspath('preprocessors'))
+
+    if args.files_path and len(args.files_path):

```

Comment:
You can skip the len checking here. Both the empty string and a string of spaces respond the same here.